src/main/scala/org/constellation/util/Rewards.scala

```diff
@@ -37,10 +40,10 @@ object Rewards {
                       )= {
     val trustEntropyMap = shannonEntropy(transitiveReputationMatrix, neighborhoodReputationMatrix)
     val distro = rewardDistribution(partitonChart, trustEntropyMap)
-    distro.mapValues(_ * rewardForEpoch(curShapshot))
+    distro.mapValues(_ * rewardDuringEpoch(curShapshot))
   }
 
-  def rewardForEpoch(curShapshot: Int) = curShapshot match {
+  def rewardDuringEpoch(curShapshot: Int) = curShapshot match {
     case num if num >= 0 && num < epochOne  => epochOneRewards
     case num if num >= 438000 && num < epochTwo => epochTwoRewards
```

Comment:
I'd also replace those hardcoded ints here. I suppose they are a leftover from when the consts like `epochOne` wheren't defined yet?

Reply:
Sure that would be great thanks!

Reply:
done